src/BenchmarkDotNet/EventHandlers/IEventHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.Results;
+using BenchmarkDotNet.Validators;
+
+namespace BenchmarkDotNet.EventHandlers
+{
+    public interface IEventHandler
+    {
+        void HandleStartValidationStage();
+        void HandleUnsupportedBenchmark(BenchmarkCase benchmarkCase);
```

Comment:
To me the **Handle**UnsupportedBenchmark name suggests that you can actually do something with the unsuported benchmark (like stop benchmarking or continue). Since it's not the case here, I think we should use the `On$Abc` rather than `Handle$Abc` naming convention.